Dockerfile

```diff
@@ -45,22 +118,16 @@ RUN mv node_modules ./migrate/node_modules
 # Copy temp node_modules of app to app folder
 RUN mv _node_modules node_modules
 
-RUN echo '#!/bin/bash\nnode /app/cli/cli.js "$@"' > /usr/bin/homarr
-RUN chmod +x /usr/bin/homarr
 RUN cd /app/cli && yarn --immutable
 
-# Expose the default application port
-EXPOSE $PORT
-ENV PORT=${PORT}
+# Root is needed for supervisord
+USER root
 
-ENV DATABASE_URL "file:/data/db.sqlite"
-ENV NEXTAUTH_URL "http://localhost:7575"
-ENV PORT 7575
-ENV NEXTAUTH_SECRET NOT_IN_USE_BECAUSE_JWTS_ARE_UNUSED
+RUN echo '#!/bin/bash\nnode /app/cli/cli.js "$@"' > /usr/bin/homarr
+RUN chmod +x /usr/bin/homarr
 
 HEALTHCHECK --interval=10s --timeout=5s --start-period=5s --retries=3 \
     CMD wget --no-verbose --tries=1 --spider http://localhost:${PORT} || exit 1
 
-VOLUME [ "/app/data/configs" ]
-VOLUME [ "/data" ]
```

Comment:
Since there was talking about reverting such changes. I do want to point out that you should avoid `VOLUME` directives here.

They create anonymous data volumes per container instance. Those will accumulate with each container created unless the container is disposed of (_eg: `docker run --rm ...` instead of just `docker run ...`_).

It is worse for Docker Compose which will be more common in usage, as destroying containers does not remove these anonymous volumes they're tied to the project and service name, thus if you did for some reason want to rely on this implicit persistence it's more difficult to reset/discard it should the need arise (_`docker volumes ls` is really unhelpful for identifying which anonymous volume belongs to what container_).

Instead it's much better to just document these locations, either in proper docs or a `compose.yaml` example (_don't use `docker-compose.yml` btw, official Docker docs are aligned with `compose.yaml` since 2023Q3 with Compose v2 release being deemed stable_).

Users that want to persist data should provide explicit config to indicate that, otherwise so long as a container is not destroyed it will persist any filesystem state it has when stopped/restarted (_sometimes a source of bugs_), whereas `VOLUME` tends to just cause more problems than benefits.